src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Set;
+import com.google.codeu.data.Datastore;
+import java.util.Arrays;
+/**
```

Comment:
nit: In order to keep it consistent with other file, you might want to add some space between the import line and the rest of the code.